libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -422,9 +429,10 @@ void AP_OSD_Screen::draw_batused(uint8_t x, uint8_t y)
 void AP_OSD_Screen::draw_message(uint8_t x, uint8_t y)
 {
     AP_Notify * notify = AP_Notify::instance();
+    uint16_t duration = message_show_time_ms * msgtime;
```

Comment:
I think you're just conventing from s to ms here? Just multiple by constant 1000 instead of using a variable